ui/lib/components/CardTable/UnselectableRow.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { getTheme } from 'office-ui-fabric-react/lib/Styling'
+import {
+  DetailsRow,
+  IDetailsListProps,
+  IDetailsRowStyles,
+  IDetailsRowProps,
+} from 'office-ui-fabric-react'
+
+const theme = getTheme()
+
+export const createUnselectableRow = (
+  when: (props: IDetailsRowProps) => boolean
+) => {
+  const renderRow: IDetailsListProps['onRenderRow'] = (props) => {
+    if (!props) {
+      return null
+    }
+
+    const customStyles: Partial<IDetailsRowStyles> = {}
+    if (when(props)) {
+      customStyles.root = {
+        backgroundColor: theme.palette.neutralLighter,
+        cursor: 'not-allowed',
+        pointerEvents: 'none',
+        color: '#aaa',
+        fontStyle: 'italic',
```

Comment:
They are not related with `UnselectableRow`.

Reply:
How about keep pointerEvents: 'none' in `UnselectableRow`, and custom other styles from `UnselectableRow` props.

Reply:
I do worry when you have pointerEvents: none then things like hovering tooltips will not work?

Reply:
You're right. I should use selection's mechanics.